util/missingmacro.txt

```diff
@@ -214,3 +214,4 @@ X509V3_set_ctx_test
 X509V3_set_ctx_nodb
 EXT_BITSTRING
 EXT_IA5STRING
+TS_VERIFY_CTS_set_certs
```

Comment:
It took me while to find out the reason for the error: the misspelled compatibility macro belongs into 'missingmacro.txt', not in 'missingcrypto.txt'. 

I reverted the 'missing*111.txt' files an force-pushed the changes to your GitHub clone. To update your local copy, you can do the following:

```
git fetch --all
git checkout signature-typo-fix 
git reset --hard origin/signature-typo-fix
```

(assuming your remote GitHub clone is referred to as 'origin'. If not, you need to change the name accordingly)

Reply:
I don't think we should ever add anything to the missing*.txt files. They are intended to be a list of functions/macros that historically never had any documentation. If we add new macros/functions they should be documented and therefore have no reason to be added to these files.

Reply:
It's only a temporary solution to fix a chicken-egg problem. @InfoHunter already volunteered to write the documentation (see #9838), but this pull request needs to be merged first. I will take care that #9838 removes the entry again.

See also our previous discussion https://github.com/openssl/openssl/pull/9472#discussion_r319386886.

Reply:
An alternative would be that @InfoHunter rebases #9838 on top of #9472 and we merge both changes together under #9838. Then we would not need the temporary addition to the 'missing*' files.

Reply:
Ah...ok then. If it is considered a temporary thing I have no objection.